sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -322,14 +361,17 @@ def convert_feature(fx):
         fig.clear()
         ax = fig.gca()
 
-    display = PartialDependenceDisplay(pd_results, features, feature_names,
-                                       target_idx, pdp_lim, deciles)
+    display = PartialDependenceDisplay(
+        pd_results, features, feature_names, target_idx,
+        pdp_lim, deciles, individual=individual, subsample=subsample
+    )
     return display.plot(ax=ax, n_cols=n_cols, line_kw=line_kw,
                         contour_kw=contour_kw)
 
 
 class PartialDependenceDisplay:
-    """Partial Dependence Plot (PDP) visualization.
+    """Partial Dependence Plot (PDP) and Individual Conditional Expectation
+    (ICE) visualization.
```

Comment:
Please try to restrict to a onliner summary followed by blank line and then expanded information to better display in IDE tooltips / API doc:

```suggestion
class PartialDependenceDisplay:
    """Partial Dependence Plot (PDP)
    
    This can also display individual partial dependencies which are often referred to
    as: Individual Condition Expectation (ICE).
    
    ...
    """

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/25febe81c3ec45e8860d524aa68b5a78c99554c9